### DISCUSSION TITLE
I am not able to create any repository either is is public or private repository

### DISCUSSION THREAD

**Discussion by Openskale** [community/community#60467] [Repositories]
### Select Topic Area

Question

### Body

I am facing this issue while creating the repository
// 20230710080258
// https://github.com/repositories

{
  "data": {
    "error": "Repository creation failed."
  },
  "code": 422
}

**Comment by rakibul-islam-hasib** [comment]
You can try another browser.

**Comment by victor-villca** [comment]
Why don't you make sure the repository name isn't already used in another one you own to avoid conflicts. Sometimes that is the problem.

**Comment by JoAzar** [comment]
What is the name of your repo?
